packages/selinux-policy/subject.cil

```diff
@@ -68,12 +68,18 @@
 (roletype system_r super_t)
 (context admin (system_u system_r super_t s0))
 
+; Processes that should never exist.
+(type forbidden_t)
+(roletype system_r forbidden_t)
+(context forbidden (system_u system_r forbidden_t s0))
+
 ; The set of all subjects.
 (typeattribute all_s)
 (typeattributeset all_s (
   kernel_t init_t system_t mount_t api_t
   network_t clock_t bus_t runtime_t
-  container_t control_t super_t))
+  container_t control_t super_t
+  forbidden_t))
```

Comment:
Add `forbidden_t` to the set of all subjects (`all_s`) since there are a couple "neverallow" rules that should apply to it also.